pkg/render/node.go

```diff
@@ -819,6 +824,11 @@ func (c *nodeComponent) clusterAdminClusterRoleBinding() *rbacv1.ClusterRoleBind
 				Name:      CalicoNodeObjectName,
 				Namespace: common.CalicoNamespace,
 			},
+			{
+				Kind:      "ServiceAccount",
```

Comment:
Is this needed for the migration or is this unrelated to the migration issue?
If it is needed for the migration then we should conditionally add this when NamespaceMigration is true.
If this isn't related to the migration I would suggest splitting it to another PR or at least in a separate commit so if we do revert the NamespaceMigration changes in the future we don't revert this accidentally.

Reply:
It is needed during the migration for the new service account, but it is also needed for operator installation in the future. It's been split into it's own commit as you recommended

Reply:
I'm not sure I understand this change either (mostly because I'm not familiar with what this binding does).

Could you explain why this is needed long-term for the CNI plugin?

Reply:
The calico-cni-plugin accesses directories on the container host, without this role it's not allowed to do so on MKE.

Reply:
Hm. CNI plugin has needed to access directories on the host for a long long time. Have we needed this all of that time and it has just never worked? Or has something recently changed that resulted in needing this permission?

Reply:
OH nevermind, it previously used the calico-node serviceaccount and so it just worked. Got it!